corona-warn-app/cwa-app-android
# Basic poster template download and caching (EXPOSUREAPP-5842)

Implements basic poster template download and caching. 

Details can be found in the first part of Point 1 from [TechSpec Poster Generation](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-client.md#organizer-poster-generation). 

Not included
- In case we receive an invalid network response or the user is offline, we should return the most recent cached or the default poster. This will be implemented in SubTask EXPOSUREAPP-6079.

How to Test
- Download latest `test_environment.json` from wiki that includes `QRCODE_POSTER_TEMPLATE_URL`
- Start MockServer
- TestMenu => EventReg => QR Code and Poster Generation
- Click "Download Poster Template" Button
- The result is shown below the QR code
